site/docs/v1/tech/apis/_reactor-response-body.adoc

```diff
@@ -0,0 +1,69 @@
+==== Response Body
+
+[.api]
+[field]#metrics.breachedPasswordMetrics# [type]#[Map<UUID, List<Object>]#::
+The breached password metrics. The map key is the Tenant Id.
+
+[field]#metrics.breachedPasswordMetrics``[tenantId]``.actionRequired# [type]#[Integer]#::
+The total number of user accounts where action is required.
+
+[field]#metrics.breachedPasswordMetrics``[tenantId]``.matchedCommonPasswordCount# [type]#[Integer]#::
+The total number of matched user accounts with a commonly compromised password.
+
+[field]#metrics.breachedPasswordMetrics``[tenantId]``.matchedExactCount# [type]#[Integer]#::
+The total number of matched user accounts with a compromised password exact match.
+
+[field]#metrics.breachedPasswordMetrics``[tenantId]``.matchedPasswordCount# [type]#[Integer]#::
+The total number of user accounts with matched passwords.
+
+[field]#metrics.breachedPasswordMetrics``[tenantId]``.matchedSubAddressCount# [type]#[Integer]#::
+The total number of sub-email user accounts that match a compromised password.
+
+[field]#metrics.breachedPasswordMetrics``[tenantId]``.passwordsCheckedCount# [type]#[Integer]#::
+The total number of user account passwords checked.
+
+[field]#status.advancedIdentityProviders# [type]#[String]#::
+The status for Advanced Identity Providers.
++
+include::_reactor-status-values.adoc[]
+
+[field]#status.advancedMultiFactorAuthentication# [type]#[String]#::
+The status for link:/docs/v1/tech/guides/multi-factor-authentication[Advanced Multi-Factor Authentication].
++
+include::_reactor-status-values.adoc[]
+
+[field]#status.advancedRegistration# [type]#[String]#::
+The status for link:/docs/v1/tech/guides/advanced-registration-forms[Advanced Registration Forms].
++
+include::_reactor-status-values.adoc[]
+
+[field]#status.applicationThemes# [type]#[String]#::
+The status for Application Themes.
++
+include::_reactor-status-values.adoc[]
+
+[field]#status.breachedPasswordDetection# [type]#[String]#::
+The status for link:/docs/v1/tech/guides/breached-password-detection[Breached Password Detection].
++
+include::_reactor-status-values.adoc[]
+* `PENDING` - the feature is pending while waiting to complete key exchange with Reactor
```

Comment:
These should be in alphabetical order, so you can either inline, create a sep include file, or use an ifdef and include the pending and disconnected values only when an asciidoctor variable is set.